src/main/java/org/folio/rest/impl/StorageHelper.java

```diff
@@ -420,9 +420,9 @@ protected static String getTenant(Map<String, String> headers) {
     return TenantTool.calculateTenantId(headers.get(RestVerticle.OKAPI_HEADER_TENANT));
   }
 
-  protected static Criterion idCriterion(String id, String schemaPath, Handler<AsyncResult<Response>> asyncResultHandler) {
+  protected static Criterion idCriterion(String id, Handler<AsyncResult<Response>> asyncResultHandler) {
     try {
-      Criteria criteria = new Criteria(schemaPath);
```

Comment:
What is the impact of not using the schema? Does it change any validation that is done? Or is it only accommodating a change in the underlying framework that doesn't make a difference in this circumstance?

Reply:
Doesn't make a difference because mod-inventory-storage doesn't have numeric fields. https://issues.folio.org/browse/CQLPG-47